OADP-4594: Update OADP channel documentation (#79577)
openshift/openshift-docs: modules/about-oadp-update-channels.adoc

:_mod-docs-content-type: CONCEPT
[id="about-oadp-update-channels_{context}"]
= About {oadp-short} update channels

When you install an {oadp-short} Operator, you choose an _update channel_. This channel determines which upgrades to the {oadp-short} Operator and to Velero you receive. You can switch channels at any time.

The following update channels are available:

* The *stable* channel is now deprecated. The *stable* channel contains the patches (z-stream updates) of OADP `ClusterServiceVersion` for `{oadp-short}.v1.1.z` and older versions from `{oadp-short}.v1.0.z`.

* The *stable-1.0* channel is deprecated and is not supported.

* The *stable-1.1* channel is deprecated and is not supported.

> * The *stable-1.2* channel contains `{oadp-short}.v1.2._z_`, the most recent {oadp-short} 1.2 `ClusterServiceVersion`.

Reviewer: OADP-1.2 is also deprecrecated and NOT supported.

> * The *stable-1.2* channel contains `{oadp-short}.v1.2._z_`, the most recent {oadp-short} 1.2 `ClusterServiceVersion`.

Author: ```suggestion
* The *stable-1.2* channel  is deprecated and is not supported.
```

* The *stable-1.3* channel contains `{oadp-short}.v1.3._z_`, the most recent {oadp-short} 1.3 `ClusterServiceVersion`.

* The *stable-1.4* channel contains `{oadp-short}.v1.4._z_`, the most recent {oadp-short} 1.4 `ClusterServiceVersion`.

For more information, see link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles].

*Which update channel is right for you?*

> * The *stable* channel is now deprecated. If you are already using the stable channel, you will continue to get updates from `{oadp-short}.v1.1._z_`.

Reviewer 2: Is the italic formatting necessary?

Reviewer 2: Global comment.

> * The *stable* channel is now deprecated.  If you are already using the stable channel, you will continue to get updates from `{oadp-short}.v1.1._z_`.

Author: ```suggestion
* The *stable* channel is now deprecated. If you are already using the stable channel, you will continue to get updates from `{oadp-short}.v1.1._z_`.
```

* Choose the *stable-1._y_* update channel to install {oadp-short} 1._y_ and to continue receiving patches for it. If you choose this channel, you will receive all z-stream patches for version 1._y_._z_.

*When must you switch update channels?*

> * If you have {oadp-short} 1._y_ installed, and you want to receive patches only for that y-stream, you must switch from the *stable* update channel to the *stable-1._y_* update channel. You will then receive all z-stream patches for version 1._y_._z_.

Reviewer 2: I personally find the plain text, italic formatting, and bold formatting distracting. I'm not sure what each style means. Some content might lose priority. Probably best to stick to one style.

* If you have {oadp-short} 1.0 installed, want to upgrade to {oadp-short} 1.1, and then receive patches only for {oadp-short} 1.1, you must switch from the *stable-1.0* update channel to the *stable-1.1* update channel. You will then receive all z-stream patches for version 1.1._z_.

> * If you have {oadp-short} 1._y_ installed, with _y_ greater than 0, and want to switch to {oadp-short} 1.0, you must _uninstall_ your {oadp-short} Operator and then reinstall it using the *stable-1.0* update channel. You will then receive all z-stream patches for version 1.0._z_.

Reviewer 2: Not sure why "uninstall" is in italics. Just to render the verb as plain text.
